packages/core/src/flow/loopUntil.ts

```diff
@@ -0,0 +1,41 @@
+import {useScene} from '../utils';
+import {decorate, threadable} from '../decorators';
+import {ThreadGenerator} from '../threading';
+import {LoopCallback} from './loop';
+
+decorate(loopUntil, threadable());
+/**
+ * Run the given generator until an event occurs.
+ *
+ * @example
+ * ```ts
+ * yield* loopUntil(
+ *   'vibrate',
+ *   () => circle().position.x(-10, 0.1).to(10, 0.1)
+ * );
+ * ```
+ *
+ * @param event - The event.
+ * @param tasks - A list of tasks to run.
```

Comment:
This parameter description is incorrect. It parameter should be called `factory` instead of `tasks`. It's also not a list of tasks to run but a function creating the generator to run. This was probably incorrectly copy&pasted from the doc block of `all`.

I'd suggest using the description of `factory` from the `loop` function.